bootstrap/090_post_setup.sql

```diff
@@ -170,22 +170,49 @@ CREATE OR REPLACE TASK TASKS.COST_CONTROL_MONITORING
     AS
 DECLARE
     sql STRING;
-    start_time timestamp_ltz default current_timestamp();
+    start_time timestamp default current_timestamp();
 BEGIN
     SYSTEM$LOG_DEBUG('starting task to monitor user and role cost');
-    -- Get and run the query to compute the daily quota usage by user and role.
-    call internal.get_daily_quota_select() into :sql;
+
+    -- Update internal.aggregated_hourly_quota with the latest usage data from the QH table function.
+    CALL INTERNAL.REFRESH_HOURLY_QUERY_USAGE_SQL(:start_time) into :sql;
```

Comment:
i don't think it makes sense to use query generation here. none of the actual query executed needs to be dynamically generated. as best i can tell, a simple udtf would suffice for the core sql and then the merge could just be here as I think it is pretty simple. We should try to avoid dynamic sql unless we have to use it.

Reply:
> none of the actual query executed needs to be dynamically generated

You cannot query the information_schema.query_history in an owner's rights stored procedure because Snowflake fail. Frustrating :\

Reply:
Yes, understood. My comment stands. Let's avoid dynamic sql unless necessary. You're patterning after how probes work but probes must do dynamic sql. I don't see that same necessity here. You're right that we can't create a udtf except from a task context. I remember this from before. That being said, it still is best to avoid dynamic sql.

Reply:
Alright, thanks for the feedback. Let me try it out.